web/packages/hovercards/src/core.ts

```diff
@@ -815,10 +820,20 @@ export default class Hovercards {
 								break;
 						}
 
+						const additionalMessage =
+							code === 404
+								? `
+								<i class="gravatar-hovercard__error-message gravatar-hovercard__error-message--claim-gravatar">
+									${ __t( this._i18n, 'Is this you?' ) } 
+									<a href="http://gravatar.com/signup" target="_blank">${ __t( this._i18n, 'Claim your free profile.' ) }</a>
```

Comment:
Let's add something that we can use to track these clicks. `utm_source`?

Reply:
@jom Good point, I have added it, thanks!